modules/ROOT/pages/js-movie-app.adoc

```diff
@@ -235,15 +237,42 @@ Click on a cast or crew member to see that person’s profile, which includes bi
 
 === Genres
 
-// TK go step by step through how to create the Genres Endpoint 
+Diving down from route to model, a call from the user would travel through the route set in `api/app.js`
+
+.api/app.js
+[source,javascript,role=nocopy]
+----
+api.get("/genres", routes.genres.list);
+---- 
+
+The information from the call would be routed by `api/routes/genres.js` in and out of the the Genre model in `api/models/genres.js`, sending the response, an array of `Genres`, back to the user. 
+
+.api/models/genres.js
+[source,javascript,role=nocopy]
+----
+var _ = require('lodash');
```

Comment:
nit: all `var`s could be replaced with `const`s

Reply:
done